### DISCUSSION TITLE
How to download the NRed kext or build it myself?

### DISCUSSION THREAD

**Discussion by aleelmaitro** [ChefKissInc/NootedRed#62] [Q&A] [answered]
Pardon my ignorance...
How is the project compiled to generate the KEXT?

**Comment by Core-i99** [comment]
using xcode
See actions for compiled builds. https://github.com/NootInc/NootedRed/actions

**Comment by NyanCatTW1** [comment, marked as answer]
You need to clone the repo recursively and have a build of Lilu.kext from https://github.com/acidanthera/Lilu/releases placed at the project root. After that, build it like any other kext project using Xcode or xcodebuild.
Alternatively, grab a prebuilt kext from https://github.com/NootInc/NootedRed/actions. You need to be logged into GitHub to see the download links. **Choose builds from the master branch unless a developer tells you to try different branches.**
